### PR TITLE
docs: Replace Paragraph with Text anywhere

### PR DESCRIPTION
**Pull request by fabriziocucci**
### Summary

Migrate V5 doc to the new MD3 typography (`Paragraph` only).

### Note

Happy to migrate the rest of the doc if this makes sense.

### CODE REVIEW

File: src/components/Card/Card.tsx [hunk 1]
```diff
@@ -112,7 +112,7 @@ export type Props = React.ComponentProps<typeof Surface> & {
  *     <Card.Title title="Card Title" subtitle="Card Subtitle" left={LeftContent} />
  *     <Card.Content>
  *       <Title>Card title</Title>
```
Comment (lukewalczak) [inline src/components/Card/Card.tsx:114]: Could you please replace the `Title` as well?

File: src/components/Card/CardContent.tsx [hunk 2]
```diff
@@ -34,13 +34,13 @@ export type Props = React.ComponentPropsWithRef<typeof View> & {
...
  * import * as React from 'react';
- * import { Card, Title, Paragraph } from 'react-native-paper';
+ * import { Card, Text, Title } from 'react-native-paper';
  *
  * const MyComponent = () => (
  *   <Card>
  *     <Card.Content>
  *       <Title>Card title</Title>
- *       <Paragraph>Card content</Paragraph>
+ *       <Text variant="bodyMedium">Card content</Text>
```
Comment (lukewalczak) [inline src/components/Card/CardContent.tsx:43]: Could you please replace the `Title` here as well?

**Comment by fabriziocucci** [comment]
Comments addressed.

**Review by lukewalczak — APPROVED** [review]

File: docs/pages/8.theming-with-react-navigation.md [hunk 3]
```diff
@@ -86,7 +86,7 @@ const HomeScreen = ({ navigation }) => (
     <Card>
       <Card.Content>
         <Title>{title}</Title>
```
[the remark this answers is not in the document]
Comment (fabriziocucci) [reply docs/pages/8.theming-with-react-navigation.md:88]: I was gonna do another PR for the other non-`Paragraph` types but I can handle this one here if you prefer! :)

[the remark this answers is not in the document]
Comment (lukewalczak) [reply docs/pages/8.theming-with-react-navigation.md]: > I was gonna do another PR for the other non-Paragraph types

Sounds good, thanks!

**Pull request merged**